warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -84,19 +84,15 @@ public static ActiveQueryLog getInstance() {
      *            the associated name by which to look up the desired {@link ActiveQueryLog}. This will typically be the name of a table or query logic.
      * @return the existing or new {@link ActiveQueryLog} for the name
      */
-    public static ActiveQueryLog getInstance(String name) {
+    public static synchronized ActiveQueryLog getInstance(String name) {
         // Return the default instance if the name is blank.
         if (StringUtils.isBlank(name)) {
             name = DEFAULT_NAME;
         }
         
         // Initialize the log cache if necessary.
         if (ActiveQueryLog.logCache == null) {
-            synchronized (ActiveQueryLog.logCacheLock) {
```

Comment:
Same comment about null check before and after synchronized potentially blocking less.